tests/Lib/DocumentCollectionTest.php

```diff
@@ -56,11 +57,11 @@ public function getIterator()
         $documents = $this->createDocuments();
         $collection = $this->element->buildFromArray($documents);
         foreach ($collection as $key => $document) {
-            self::assertTrue($document instanceof Lib\Document);
+            self::assertTrue($document instanceof Document);
         }
     }
 
-    /**
+    /**searchtest
```

Comment:
:+1: It looks very much like it. Perhaps I was considering using the @group annotation. Either way, it's going :)